lib/capybara/node/matchers.rb

```diff
@@ -350,10 +350,10 @@ def has_no_button?(locator, options={})
       #
       # Note: 'textarea' and 'select' are valid type values, matching the associated tag names.
       #
-      # @param [String] locator           The label, name or id of a field to check for
-      # @option options [String] :with    The text content of the field
-      # @option options [String] :type    The type attribute of the field
-      # @return [Boolean]                 Whether it exists
+      # @param [String] locator                  The label, name or id of a field to check for
+      # @option options [String/Regexp] :with    The text content of the field or a Regexp to match
```

Comment:
In Yard a types specifier list is a comma separated list of types, not / separated